scripts/neural_machine_translation/filter_langs_nmt.py

```diff
@@ -20,8 +20,16 @@
 import warnings
 from pathlib import Path
 from time import sleep
-
-import fasttext
+import os
+
+try:
+    import fasttext
+    # temp fix for the warning messages
+    fasttext.FastText.eprint = lambda x: None
+except ImportError:
```

Comment:
I think this should be fixed by adding fastText to the nlp specific requirements.txt file rather than installing here.